packages/tool-cache/src/tool-cache.ts

```diff
@@ -30,12 +30,13 @@ const userAgent = 'actions/tool-cache'
  *
  * @param url       url of tool to download
  * @param dest      path to download tool
+ * @param auth      authorization header
  * @returns         path to downloaded tool
  */
 export async function downloadTool(
   url: string,
   dest?: string,
-  token?: string
+  auth?: string
```

Comment:
Nit pick change since downloadTool is generic (shouldn't assume `token` auth scheme)